CMakeLists.txt

```diff
@@ -16,12 +16,40 @@ ExternalProject_Add(googletest
   LOG_BUILD         ON
 )
 
+include("cmake/define_option.cmake")
+
+define_option(ROC_INCLUDE_DIR "" STRING "roc toolkit include directory")
+define_option(ROC_LIB_DIR "" STRING "roc toolkit library directory")
+define_option(DOWNLOAD_ROC ON BOOL "automatically download and build roc-toolkit")
+
+if(DOWNLOAD_ROC)
+  include("cmake/download_roc.cmake")
+else()
+  if(NOT ROC_INCLUDE_DIR STREQUAL "")
+    get_filename_component(ROC_INCLUDE_DIR "${ROC_INCLUDE_DIR}" ABSOLUTE)
+    message(STATUS "Using ROC_INCLUDE_DIR - ${ROC_INCLUDE_DIR}")
+
+    include_directories(SYSTEM "${ROC_INCLUDE_DIR}")
+  endif()
+
+  if(NOT ROC_LIB_DIR STREQUAL "")
+    get_filename_component(ROC_LIB_DIR "${ROC_LIB_DIR}" ABSOLUTE)
+    message(STATUS "Using ROC_LIB_DIR - ${ROC_LIB_DIR}")
+
+    link_directories("${ROC_LIB_DIR}")
+  endif()
+
+  link_libraries("roc")
+endif()
+
 include_directories(
   "${CMAKE_CURRENT_BINARY_DIR}/googletest-src/googletest/include"
+  #"${CMAKE_CURRENT_BINARY_DIR}/roc-toolkit-src/src/public_api/include"
 )
 
 link_directories(
   "${CMAKE_CURRENT_BINARY_DIR}/googletest-build/lib"
+  #"${CMAKE_CURRENT_BINARY_DIR}/roc-toolkit-src/bin/x86_64-pc-linux-gnu/"
 )
```

Comment:
Please remove commented code.

Reply:
Fixed and moved to `cmake/download_googletest.cmake`